Add cflags and ldflags for log4cxx, found with pkg-config. Not sure where else
to get those flags with ros* tools.

--- tools/rosconsole/manifest.xml~	2010-11-10 00:51:22.000000000 +0100
+++ tools/rosconsole/manifest.xml	2011-01-16 16:37:30.000000000 +0100
@@ -6,7 +6,7 @@
   <url>http://www.ros.org/wiki/rosconsole</url>
   <depend package="rostime"/>
   <export>
-    <cpp cflags="-I${prefix}/include `rosboost-cfg --cflags`" lflags="-Wl,-rpath,${prefix}/lib -L${prefix}/lib -lrosconsole `rosboost-cfg --lflags thread` -llog4cxx"/>
+    <cpp cflags="-I${prefix}/include `rosboost-cfg --cflags` `pkg-config liblog4cxx --cflags`" lflags="-Wl,-rpath,${prefix}/lib -L${prefix}/lib -lrosconsole `rosboost-cfg --lflags thread` `pkg-config liblog4cxx --libs`"/>
   </export>
   <rosdep name="apr"/>
   <rosdep name="log4cxx"/>
